### PR TITLE
RAC-4023: retire old rackhd-package.yml and merge similar function ansible yaml

### PR DESCRIPTION
**Pull request by panpan0000**
1. remove old bad rackhd-package.yml (it doesn't have the version control on deb packages)
2. use passing in deb-repository instead of using two rackhd_package.yml and rackhd_release.yml to use released deb source or dev-build deb source .
    2.a rackhd_ci_build.yml & rackhd_release.yml have been merged and replace rackhd_package.yml
    2.b rackhd_ci_build_mini.yml & rackhd_release_mini.yml have been merged and replace rackhd_package_mini.yml

test done:
image

### CODE REVIEW

File: packer/HWIMO-BUILD [hunk 1]
```diff
@@ -119,6 +120,19 @@ else
    fi
 fi
 
+### Choose a deb repository/component , according to IS_OFFICIAL_RELEASE parameter
+if [ $ANSIBLE_PLAYBOOK != "rackhd_local" ]; then
+       if [ ! -n "${DEBIAN_REPOSITORY}" ];  then
+           DEBIAN_REPOSITORY="deb https://dl.bintray.com/rackhd/debian trusty release"
```
Comment (changev) [inline packer/HWIMO-BUILD]: There's default value in playbook so why set again here?

Comment (panpan0000) [reply packer/HWIMO-BUILD]: fixed

**Comment by PengTian0** [comment]
👍

**Comment by changev** [comment]
+1

**Pull request merged**